pkg/controller/container-runtime-config/helpers.go

```diff
@@ -28,10 +27,14 @@ import (
 const (
 	minLogSize           = 8192
 	minPidsLimit         = 20
-	crioConfigPath       = "/etc/crio/crio.conf"
 	storageConfigPath    = "/etc/containers/storage.conf"
 	registriesConfigPath = "/etc/containers/registries.conf"
 	policyConfigPath     = "/etc/containers/policy.json"
+	// CRIODropInFilePathPidsLimit is the path at which changes to the crio config for pids-limit
+	// will be dropped in this is exported so that we can use it in the e2e-tests
+	CRIODropInFilePathPidsLimit  = "/etc/crio/crio.conf.d/02-ctrcfg-pidsLimit"
+	crioDropInFilePathLogLevel   = "/etc/crio/crio.conf.d/01-ctrcfg-logLevel"
+	crioDropInFilePathLogSizeMax = "/etc/crio/crio.conf.d/03-ctrcfg-logSizeMax"
```

Comment:
nit: I'd probably have these the same priority `01`, having them have different implies (to me) they're of different importance

Reply:
fixed